src/Compression/CompressionCodecSZ3.cpp

```diff
@@ -196,8 +198,9 @@ void CompressionCodecSZ3::setAndCheckVectorDimension(size_t dimension_)
 
 void CompressionCodecSZ3::doDecompressData(const char * source, UInt32 source_size, char * dest, UInt32 /*uncompressed_size*/) const
 {
-    if (source_size == 0)
-        throw Exception(ErrorCodes::CORRUPTED_DATA, "Can not decompress empty data");
+    static constexpr const size_t config_size = 39;
```

Comment:
```suggestion
    /// Hardcoded, because it is not declared (we just calculated the minimal size of decompressed config) and it is less than sizeof(SZ3::Config).
    static constexpr const size_t config_size = 39;
```